http_client.rst

```diff
@@ -995,9 +995,13 @@ Enabling cURL Support
 ~~~~~~~~~~~~~~~~~~~~~
 
 This component can make HTTP requests using native PHP streams and the
-``amphp/http-client`` and cURL libraries. Although they are interchangeable and
-provide the same features, including concurrent requests, HTTP/2 is only supported
-when using cURL or ``amphp/http-client``.
+``amphp/http-client`` (version 5.0 or higher) and cURL libraries. Although they
```

Comment:
Based on https://github.com/symfony/symfony/pull/60734, I suggest mentioning 5.3.2 as lowest supported version instead of 5.0